src/app/shared/dialog-box/dialog-box.component.scss

```diff
@@ -0,0 +1,18 @@
+textarea {
+  width: 80% !important;
```

Comment:
is that needed? Can we rely on govuk lib solely please?
If we think the design are not matching the design system we can have a chat about it.
The components from the design system lib should be ready to be used.
We never be using !important or inline/js style in components